sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/receipt/__init__.py

```diff
@@ -0,0 +1,11 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+
+from ._receipt_verification import verify_receipt
+from ._version import VERSION
+
+__version__ = VERSION
```

Comment:
We don't need a `._version.py` file in this as it's not shipping as a separate package and wont version independently.
(So lines 8 and 10 should be removable)

Reply:
I was following the same example of the `certificate` submodule in the same library (see [comment above](https://github.com/Azure/azure-sdk-for-python/pull/27265#discussion_r1064458444)), but indeed a new version seems unnecessary. Will remove it.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-python/pull/27265/commits/e558b7840ceebc54673364d5886280c06a7e1ad1